src/bin/server.cpp

```diff
@@ -10,6 +10,46 @@
 #include "tpcds/tpcds_table_generator.hpp"
 #include "tpch/tpch_table_generator.hpp"
 
+namespace {
+
+void generate_benchmark_data(std::string argument_string) {
+  std::vector<std::string> benchmark_data_config;
+
+  // Remove unnecessary white spaces
+  boost::trim_if(argument_string, boost::is_any_of(":"));
+
+  // Remove dashes and convert to lower case to unify different writings of benchmarks ("TPC-H", "tpch", or "tpc-h")
+  boost::replace_all(argument_string, "-", "");
+  boost::to_lower(argument_string);
+
+  // Split benchmark name and sizing factor
+  boost::split(benchmark_data_config, argument_string, boost::is_any_of(":"), boost::token_compress_on);
+  Assert(benchmark_data_config.size() < 3,
+         "Malformed input for benchmark data generation. Expecting a benchmark "
+         "name and a sizing factor.");
```

Comment:
You are listing exactly two things - why are you comparing against 3?

Reply:
Oh sorry, leftover from `size` being optional for the console.